retroshare-gui/src/util/HandleRichText.cpp

```diff
@@ -621,10 +621,10 @@ QString RsHtml::formatText(QTextDocument *textDocument, const QString &text, ulo
 	QDomDocument doc;
 	if (doc.setContent(formattedText, &errorMsg, &errorLine, &errorColumn) == false) {
 
-		// convert text with QTextDocument
-		QTextDocument textDoc;
-		textDoc.setPlainText(text);
-		formattedText=textDoc.toHtml();
+		// convert text with QTextBrowser
+		QTextBrowser textBrowser;
```

Comment:
QTextBrowser needs to be in GUI thread... That why this change.
I've posted in lobby a fix but doesn't get reports.

Reply:
Hi, i hear nothing from you then created this PR,
I will test then tonight your fix

Reply:
It would be a good idea to make a post in RS dev forum when one creates a PR.
Good idea also to provide a description of what the PR does.

Reply:
@PhenomRetroShare can you make a new PR for test or sent me the right patch for it, i doesnt saw your chat lobby post